File: Open Kattis/cd.py
m,n=map(int,input().split())
while(m!=0 or n!=0):
    array=[0]*m
    for i in range(m):
        p=int(input())
        array[p-1]+=1
    for i in range(n):
        p=int(input())
        if(p<=m):
            array[p-1]+=1
    con=0
    for i in array:
        if(i!=2):
            con+=1
    print(m-con)

    m,n=map(int,input().split())

File: Open Kattis/cd.cpp
#include <iostream>
#include <vector>
using namespace std;
int main(){
    int m,n,p;
    cin>>m>>n;
    while(m!=0 || n!=0){
        vector<int> array(m,0);
        for (int i=0; i<m; i++){
            cin>>p;
            array[p-1]++;
        }
        for (int i=0;i<n;i++){
            cin>>p;
            if(p<=m)
                array[p-1]++;
        }
        int con=0;
        for(int i=0;i<m;i++){
            if(array[i]!=2)
                con++;
        }
        cout<<m-con<<endl;
        cin>>m>>n;
    }
    return 0;
}